### ISSUE TITLE
The distinction between different speaker is not obvious.

### ISSUE THREAD

**Issue by chynphh** [NVIDIA/mellotron#31]
When using multi-speaker data, can not distinguish between male and female, and there are only slight differences between different speakers. Current training step is 32K. Is this normal? The language is mandarin.

**Comment by rafaelvalle** [comment]
Is the loss still going down and model is not overfitting, i.e. generalization error is not increasing?

**Comment by gongchenghhu** [comment]
I also use multi-speaker data, include half of biaobei, a private male datasets(5000 sentences), and other 4 private small datasets, totally 6 speakers. When I use this datasets to train mellotron, I can't get a good alignment, the alignment like the below figure
![image](https://user-images.githubusercontent.com/45996835/111429176-96eb0a80-8733-11eb-8f91-8ea11d404a73.png)
But while I only use the single speaker dataset bioabei, the alignment is  good.
So I want to know, what's your mandarin multi-speaker data look like, how many speakers? And 6 speakers is enough?
@chynphh
